### PR TITLE
✨Added support for iplabel RUMBI to AMP analytics

### PR DESCRIPTION
**Pull request by aannoune**
https://github.com/ampproject/amphtml/issues/15628

### CODE REVIEW

**Comment by aannoune** [comment]
I signed it!

**Comment by aannoune** [comment]
I signed it!

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -964,6 +964,59 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
...
+      '&l=${browserLanguage}',
+    },
+    'triggers': {
+      'trackPageview': {
+        'on': 'visible',
+        'request': 'onload',
+      },
+    },
+    'transport': {
+      'beacon': false,
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors.js]: just to confirm that your server doesn't support `beacon` and `xhrpost`. AMP will handle fallback case if browser doesn't support them, so it's always recommended to support all.

File: examples/analytics-vendors.amp.html [hunk 2]
```diff
@@ -541,6 +542,25 @@
...
+       "vars": {
+        "trackerId":"ampTrackerId"
+        "customdim":"",
+        "business":"",
+        "abtesting":"",
+        "infrastructure":"",
+        "customer":"",
+        "clientip":"",
+        "urlgroup":""
+           }
```
Comment (zhouyx) [inline examples/analytics-vendors.amp.html]: nit: please fix indentation. and remove empty lines.

File: examples/analytics-vendors.amp.html [hunk 3]
```diff
@@ -541,6 +542,23 @@
 </amp-analytics>
 <!-- End INFOnline example -->
 
+<!-- iplabel rumbi. Ensure you replace "ampTrackerId" with your tracker id . -->
+<amp-analytics type="iplabel" id="iplabel" >
+<script type="application/json">
+{
+  "vars": {
+   "trackerId":"ampTrackerId"
```
Comment (zhouyx) [inline examples/analytics-vendors.amp.html]: nit: Fail to include comma. Lead to json parse error.

**Comment by aannoune** [comment]
I have some trouble understanding what happend with percy and travis tests on my last commit

**Comment by zhouyx** [comment]
@aannoune I restarted the test. LGTMed and merged. Thank you for the Pull Request.

**Pull request merged**